importer/analytics-datastore-clickhouse/clickhouseTables.js

```diff
@@ -1,119 +1,119 @@
 const queries = [
   // Resource Tables
   `CREATE TABLE organization(
-		facility_id String,
-		id String,
-		version String,
+		facility_id String null,
```

Comment:
Can we please make these capitalized NULL as well to keep it consistent with the rest of the query?

Reply:
Done! Thanks